oidc-wp/openid-connect-generic
# fix(tokens): Fixes the potential of deleting a working refresh_token when no new one is available

### All Submissions:


### Changes proposed in this Pull Request:

`save_refresh_token()` disregard the existing and still valid `refresh_token` and replace it with false (since the response to a renewal does not contain it)

Closes #404

### How to test the changes in this Pull Request:

1. Use Google OIDC with refresh token
```php
        add_filter( 'openid-connect-generic-auth-url', function( string $url ) {
            return $url . '&access_type=offline&prompt=consent';
        });
```
2. Connect
3. See from your logs that, without this PR you're disconnect after 1h (the first call to`refresh_token`) 

### Other information:



### Changelog entry

Do not empty refresh_token if renewed access tokens do not come with new refresh_token.
